ClientApp/src/styles/index.scss

```
@@ -255,3 +255,11 @@ $dot-size: 1.25em;
     position: relative;
   }
 }
+
+.TableBody span {
```

`TableBody` is not a great class to apply this style to – there may be cases where we want a table body that doesn't have the overflow ellipsis behavior. My suggestion would be
```suggestion
.overflow-ellipsis,
.overflow-ellipsis span {
```
and apply the class `overflow-ellipsis`, rather than `TableBody` to the div. Also, CSS class names usually use `kebab-case` – the only reason we have some in `PascalCase` is to match the names of React components.

---

Good call-out. Thanks!